### PR TITLE
Move mutations to its own files

### PR DESCRIPTION
**Pull request by dimak1**

### CODE REVIEW

**Comment by dimak1** [comment]
Work in progress. Need to add optimistic responses, but wanted to get your feedback earlier, if any.

File: app/mutations/application/updateApplicationStatusByRowIdMutation.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  updateApplicationStatusByRowIdMutationVariables
+} from 'updateApplicationStatusByRowIdMutation.graphql';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation updateApplicationStatusByRowIdMutation(
+    $input: UpdateApplicationStatusByRowIdInput!
+  ) {
+    updateApplicationStatusByRowId(input: $input) {
+      applicationStatus {
```
Comment (dleard) [inline app/mutations/application/updateApplicationStatusByRowIdMutation.ts]: Add clientMutationId to the mutation
```suggestion
     clientMutationId
      applicationStatus {
```

File: app/mutations/product/createProductMutation.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import {
+  createProductMutation as createProductMutationType,
+  createProductMutationVariables
+} from 'createProductMutation.graphql';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation createProductMutation($input: CreateProductInput!) {
+    createProduct(input: $input) {
```
Comment (dleard) [inline app/mutations/product/createProductMutation.ts]: Same thing here. These mutations were created way before we started adding the clientMutationId
```suggestion
     clientMutationId
    createProduct(input: $input) {
```

Comment (dimak1) [reply app/mutations/product/createProductMutation.ts]: Added

File: app/mutations/application/updateApplicationStatusByRowIdMutation.ts (hunk 1 above)
Comment (dimak1) [reply app/mutations/application/updateApplicationStatusByRowIdMutation.ts]: Added

**Comment by dimak1** [comment]
@dleard I made updates. Please take a look. Thanks!

File: app/mutations/industry/editUserMutation.ts [hunk 3]
```diff
@@ -1,19 +1,38 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
 import BaseMutation from '../BaseMutation';
+import {
+  editUserMutation as editUserMutationType,
+  editUserMutationVariables
+} from '../../__generated__/editUserMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/industry/editUserMutation.ts]:
```suggestion
} from 'editUserMutation.graphql';
```
The typescript server will look for files in the `__generated__` folder, no need to add the full relative path

File: app/mutations/product/saveProductMutation.ts [hunk 4]
```diff
@@ -1,5 +1,10 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
 import BaseMutation from '../BaseMutation';
+import {
+  saveProductMutation as saveProductMutationType,
+  saveProductMutationVariables
+} from '../../__generated__/saveProductMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/product/saveProductMutation.ts]:
```suggestion
} from 'saveProductMutation.graphql';
```

File: app/mutations/user_organisation/UpdateUserOrganisation.ts [hunk 5]
```diff
@@ -1,5 +1,10 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
 import BaseMutation from '../BaseMutation';
+import {
+  UpdateUserOrganisationMutation as updateUserOrganisationMutationType,
+  UpdateUserOrganisationMutationVariables
+} from '../../__generated__/UpdateUserOrganisationMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/user_organisation/UpdateUserOrganisation.ts]:
```suggestion
} from 'UpdateUserOrganisationMutation.graphql';
```

File: app/mutations/application/updateApplicationStatusMutation.ts [hunk 6]
```diff
@@ -1,4 +1,9 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  updateApplicationStatusMutation as updateApplicationStatusMutationType,
+  updateApplicationStatusMutationVariables
+} from '../../__generated__/updateApplicationStatusMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/application/updateApplicationStatusMutation.ts]:
```suggestion
} from 'updateApplicationStatusMutation.graphql';
```

File: app/mutations/benchmark/createBenchmarkMutation.ts [hunk 7]
```diff
@@ -1,4 +1,9 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  createBenchmarkMutation as createBenchmarkMutationType,
+  createBenchmarkMutationVariables
+} from '../../__generated__/createBenchmarkMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/benchmark/createBenchmarkMutation.ts]:
```suggestion
} from 'createBenchmarkMutation.graphql';
```

File: app/mutations/benchmark/editBenchmarkMutation.ts [hunk 8]
```diff
@@ -1,5 +1,10 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
 import BaseMutation from '../BaseMutation';
+import {
+  editBenchmarkMutation as editBenchmarkMutationType,
+  editBenchmarkMutationVariables
+} from '../../__generated__/editBenchmarkMutation.graphql';
```
Comment (matthieu-foucault) [inline app/mutations/benchmark/editBenchmarkMutation.ts]:
```suggestion
} from 'editBenchmarkMutation.graphql';
```

**Pull request closed without merging**